### PR TITLE
DOC add testimonials

### PR DESCRIPTION
**Pull request by MarcoGorelli**
@girip11 is it OK if I include such a quote from you as a testimonial? If so, please let me know if you'd like your description changing / have a different website to link to

### CODE REVIEW

**Comment by girip11** [comment]
I am completely fine with this testimonial. Github link is good enough for me.

**Review by girip11 — APPROVED** [review]

**Comment by MarcoGorelli** [comment]
cool thanks, I'm just waiting for Alex's approval then'll merge

**Pull request merged**
